site/content/includes/m3dbnode/annotated_config.yaml

```diff
@@ -363,6 +363,9 @@ db:
         # Whether or not writes to leaving shards count towards consistency
         # Default = false
         shardsLeavingCountTowardsConsistency: <bool>
+        # Whether or not writes to both leaving and initializing shards as pair counts toward consistency
```

Comment:
hmm i'm thinking about the interplay between your new bool `shardsLeavingAndInitializingCountTowardsConsistency` and the old bool `shardsLeavingCountTowardsConsistency`. 

i know we don't use the latter today at all - maybe we should add a check to ensure they can't both be set at the same time tho (or that one takes priority over the other)

Reply:
i see you already have a check for it in the tests. maybe add a comment explaining the intent here too